README.md

```diff
@@ -29,6 +29,19 @@ instance. So, please, run the following command:
     git clone git://github.com/maestrodev/puppet-wget.git modules/wget
 
 
+Hiradata Config
+---------------
+
+Example configuration using Hiradata style config:
+
+```
+composer::user: 'vagrant'
+composer::group: 'vagrant'
+composer::target_dir: '/usr/local/bin/'
+composer::auto_update: true
+```
```

Comment:
when writing docs I'd like to see how to configure it using pure puppet and not with hiera. Hiera is a nice tool of course, but not everyone uses it.